src/azul/plugins/metadata/hca/aggregate.py

```diff
@@ -81,19 +83,18 @@ def _get_accumulator(self, field) -> Optional[Accumulator]:
         elif field in ('size', 'count'):
             return DistinctAccumulator(SumAccumulator())
         else:
-            return None
+            return super()._get_accumulator(field=field)
```

Comment:
```suggestion
            return super()._get_accumulator(field)
```

https://github.com/DataBiosphere/azul/blob/develop/CONTRIBUTING.rst#123method-and-function-arguments